packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -525,7 +519,7 @@ describe('Combobox', () => {
             // prettier-ignore
             const viewTemplate = html`
                 <${comboboxTag}
-                    autocomplete="inline"
+                    autocomplete="list"
```

Comment:
This change causes the test in this describe block to fail both with and without the other changes in this PR. I haven't yet investigated why: interactively it behaves correctly to scroll the selected item into view when `autocomplete="list"`.

Reply:
So, the issue is that when you change to `list` the items in the dropdown are limited to what has been typed in the input, whereas before they were not. Changing this to `none` (or simply removing this attribute), in addition to a change needed in the `focusAndScrollOptionIntoView` implementation (see this [commit](https://github.com/ni/nimble/pull/2376/commits/c53bc58288b301d6bbceb5a6d9048532d357cb31)).